automatic/brave/update.ps1

```diff
@@ -1,97 +1,89 @@
-  $domainStable = 'https://github.com'
-  # Web url was provided at https://github.com/chocolatey-community/chocolatey-packages/issues/1791#issuecomment-1030152913
-  $releaseStableVersion = Invoke-RestMethod -Uri $releaseStableUrl
-  $url32 = $domainStable + ('/brave/brave-browser/releases/download/v{0}/BraveBrowserStandaloneSilentSetup32.exe' -f $releaseStableVersion)
-  $url64 = $domainStable + ('/brave/brave-browser/releases/download/v{0}/BraveBrowserStandaloneSilentSetup.exe' -f $releaseStableVersion)
-  $version = $releaseStableVersion
-
-  $streams = @{
-    stable = @{
-      URL32         = $url32
-      URL64         = $url64
-      Version       = $version
-      RemoteVersion = $version
-      Title         = 'Brave Browser'
-      IconUrl       = 'https://cdn.jsdelivr.net/gh/chocolatey-community/chocolatey-coreteampackages@a23ca306537e2537a574ddc55e9c28dc1151ab30/icons/brave.svg'
-    }
-
-    beta   = @{
-      URL32         = $url32_b
-      URL64         = $url64_b
-      Version       = $version_b + '-beta'
-      RemoteVersion = $version_b
-      Title         = 'Brave Browser (Beta)'
-      IconUrl       = 'https://cdn.jsdelivr.net/gh/chocolatey-community/chocolatey-coreteampackages@a23ca306537e2537a574ddc55e9c28dc1151ab30/icons/brave-beta.svg'
-    }
-  }
-
-  if (!$url64 -and !$url64_b) {
-    Write-Host "No stable and no beta release is available (Nightly not supported)..."
-    return "ignore"
+$availableStreams = [ordered] @{
+  stable = @{
+    Title           = 'Brave Browser'
+    ReleaseUrl      = 'https://brave-browser-downloads.s3.brave.com/latest/release.version'
+    DownloadUrl32   = 'https://github.com/brave/brave-browser/releases/download/v{0}/BraveBrowserStandaloneSilentSetup32.exe'
+    DownloadUrl64   = 'https://github.com/brave/brave-browser/releases/download/v{0}/BraveBrowserStandaloneSilentSetup.exe'
+    VersionFormat   = '{0}'
+    IconUrl         = 'https://cdn.jsdelivr.net/gh/chocolatey-community/chocolatey-coreteampackages@a23ca306537e2537a574ddc55e9c28dc1151ab30/icons/brave.svg'
   }
-
-  # Just because we have a version returned does not mean there is a Windows version available
-  # if the URL is valid, it won't throw
-  try {
-    Invoke-RestMethod -Uri $url64 -UseBasicParsing -Method HEAD
-  }
-  catch {
-    $streams.stable = 'ignore'
-    Write-Host "No stable release is available"
+  beta = @{
+    Title           = 'Brave Browser (Beta)'
+    ReleaseUrl      = 'https://brave-browser-downloads.s3.brave.com/latest/beta.version'
+    DownloadUrl32   = 'https://github.com/brave/brave-browser/releases/download/v{0}/BraveBrowserStandaloneSilentBetaSetup32.exe'
+    DownloadUrl64   = 'https://github.com/brave/brave-browser/releases/download/v{0}/BraveBrowserStandaloneSilentBetaSetup.exe'
+    VersionFormat   = '{0}-beta'
+    IconUrl         = 'https://cdn.jsdelivr.net/gh/chocolatey-community/chocolatey-coreteampackages@a23ca306537e2537a574ddc55e9c28dc1151ab30/icons/brave-beta.svg'
```

Comment:
THese should still be located inside of the `au_GetLatest` function, not outside of it.

Please don't reorder the file, or change the logic if it isn't needed.